microsoft/onefuzz
# Add github issues integration

## Summary of the Pull Request

Adding Github Issues integration for crash reports

## PR Checklist
* [X] Applies to work item: #109 

## Info on Pull Request

Adds Github Issue integration using user-based personal access tokens.

## Validation Steps Performed

1. Create a crash reporting task.  
2. Add a notification configuration for Github Issues 
3. use `onefuzz debug notification task task_id` to generate a fake crash report for said task
4. Verify github issues are created and updated on duplicate
